### ISSUE TITLE
safe mode after last crash

### ISSUE THREAD

**Issue by meepingsnesroms** [libretro/RetroArch#2855]
On android if you set a driver incorrectly or disable overlay and touchscreen input you have to delete and reinstall retroarch or use a file manager (that most people dont have on there phones) to edit or delete /sdcard/Android/data/com.retroarch/files/retroarch.cfg

For safe mode if retroarch crashed last time it was open,display a safe mode screen and load with default settings but still display the defective settings so they can be fixed.

**Comment by micechal** [comment]
Doesn't clearing application data restore the defaults though?

**Comment by meepingsnesroms** [comment]
Yes but then you also have to download 300mb+ of cores and databases again then reconfigure everything.

**Comment by fr500** [comment]
No ETA for this.
It's hard to tell what the "deffective settings" is

**Comment by meepingsnesroms** [comment]
You don't have to know what the defective settings are just display the old settings and "Safe Mode" in the corner and the user can change whatever settings they changed last time it worked.

**Comment by fr500** [comment]
There is no separation between the GUI representation and the setting so this would be really complex to implement

**Comment by yllekz** [comment]
I ran into a problem tonight on Wii hardware running Retroarch. I loaded the Genesis Plus core which somehow completely locked up Retroarch at a black screen. Resetting the console and reloading Retroarch didn't fix it either so I had to remove the SD card and basically nuke the entire Retroarch apps folder and start from scratch.

I definitely think there needs to be an "oh s**t mode" for when this occurs so your entire Retroarch install isn't borked as a result.
